content/en/docs/tasks/configmap-secret/managing-secret-using-kubectl.md

```diff
@@ -139,6 +139,14 @@ The output is similar to:
 1f2d1e2e67df
 ```
 
+In order to avoid storing a secret encoded value in your shell history, you can run the following command:
```

Comment:
How about adding a shell comment just before:
```
echo 'MWYyZDFlMmU2N2Rm' | base64 --decode
```

to caution readers that the secret data may end up in their shell history?

Reply:
Do you mean something like:

```shell
# Warning: MWYyZDFlMmU2N2Rm will be stored in your shell history
echo 'MWYyZDFlMmU2N2Rm' | base64 --decode
```

Reply:
Done